### PR TITLE
Editorial review updates

### PR DESCRIPTION
**Pull request by mattwojo**

### CODE REVIEW

File: WSL/about.md [hunk 1]
```diff
@@ -2,15 +2,13 @@
...
-ms.custom: seodec18
-ms.localizationpriority: high;
+ms.localizationpriority: high
 ---
-# Overview of the Windows Subsytem for Linux
 
-## Windows Subsystem for Linux
+# What is the Windows Subsystem for Linux?
 
 The Windows Subsystem for Linux lets developers run a GNU/Linux environment -- including most command-line tools, utilities, and applications -- directly on Windows, unmodified, without the overhead of a virtual machine.
```
Comment (craigloewen-msft) [inline WSL/about.md:13]:
```suggestion
The Windows Subsystem for Linux lets developers run a GNU/Linux environment -- including most command-line tools, utilities, and applications -- unmodified and directly on Windows in a highly integrated environment.
```

File: WSL/about.md [hunk 2]
```diff
@@ -20,12 +18,32 @@ You can:
...
+
+Individual Linux distributions can be run with either the WSL 1 or WSL 2 architecture. Each distribution can be upgraded or downgraded at any time and you can run WSL 1 and WSL 2 distributions side by side. WSL 2 uses an entirely new architecture that benefits from running a real Linux kernel.
+
+## Next steps
+
+    - [Install WSL 1 and update to WSL 2](./install-win10.md)
+
+    - [Comparing WSL 2 and WSL 1](./compare-versions.md)
+    
+    - [ ]()
```
Comment (craigloewen-msft) [inline WSL/about.md:41]: We can't merge these into master without this table being filled out or removed.

File: WSL/about.md [hunk 3]
```diff
@@ -20,12 +18,32 @@ You can:
 * Run common command-line tools such as `grep`, `sed`, `awk`, or other ELF-64 binaries.
 * Run Bash shell scripts and GNU/Linux command-line applications including:  
     *Tools: vim, emacs, tmux
```
Comment (craigloewen-msft) [inline WSL/about.md:20]:
```suggestion
    * Tools: vim, emacs, tmux
```

File: WSL/about.md [hunk 4]
```diff
@@ -20,12 +18,32 @@ You can:
...
-    *Languages: Javascript, Ruby, Python, C/C++, C# & F#, Rust, Go, etc.
-    *Services: SSHD, MySQL, Apache, lighttpd
+    *Languages: [NodeJS](https://docs.microsoft.com/windows/nodejs/setup-on-wsl2), Javascript, [Python](https://docs.microsoft.com/windows/python/web-frameworks), Ruby, C/C++, C# & F#, Rust, Go, etc.
```
Comment (craigloewen-msft) [inline WSL/about.md:21]:
```suggestion
    * Languages: [NodeJS](https://docs.microsoft.com/windows/nodejs/setup-on-wsl2), Javascript, [Python](https://docs.microsoft.com/windows/python/web-frameworks), Ruby, C/C++, C# & F#, Rust, Go, etc.
```

File: WSL/about.md [hunk 5]
```diff
@@ -20,12 +18,32 @@ You can:
...
+    *Services: SSHD, MySQL, Apache, lighttpd, [MongoDB](https://docs.microsoft.com/windows/nodejs/databases), [PostgreSQL](https://docs.microsoft.com/windows/python/databases).
```
Comment (craigloewen-msft) [inline WSL/about.md:22]:
```suggestion
    * Services: SSHD, MySQL, Apache, lighttpd, [MongoDB](https://docs.microsoft.com/windows/nodejs/databases), [PostgreSQL](https://docs.microsoft.com/windows/python/databases).
```

File: WSL/compare-versions.md [hunk 6]
```diff
@@ -0,0 +1,163 @@
...
+
+When using a WSL 1 distribution, if your computer was set up to be accessed by your LAN, then applications run in WSL could be accessed on your LAN as well.
+
+This isn't the default case in WSL 2. WSL 2 has a virtualized ethernet adapter with its own unique IP address. Currently, to enable this workflow you will need to go through the same steps as you would for a regular virtual machine. (We are looking into ways to improve this experience.)
+
+#### IPv6 access
+
+WSL 2 distributions currently cannot reach IPv6-only addresses. We are working on adding this feature.
+
+## WSL 2 Virtual Hardware Disk limitations
```
Comment (craigloewen-msft) [inline WSL/compare-versions.md:130]: Consider changing this H2 to be more positive.

```suggestion
## How to expand the WSL 2 virtual hard disk
```

File: WSL/file-permissions.md [hunk 7]
```diff
@@ -54,11 +53,11 @@ If the file has metadata present, we simply use those metadata values instead of
 
 The result depends on if the file already has existing metadata.
 
-##### **The file does not have metadata (default)**
+##### chmod file does not have metadata (default)
```
Comment (craigloewen-msft) [inline WSL/file-permissions.md:56]: I'd revert this one back. To be accurate it's the file that you are using 'chmod' on. Perhaps another wording could be clearer here.

```suggestion
##### The file does not have metadata (default)

```

File: WSL/file-permissions.md [hunk 8]
```diff
@@ -54,11 +53,11 @@ If the file has metadata present, we simply use those metadata values instead of
 
 The result depends on if the file already has existing metadata.
 
-##### **The file does not have metadata (default)**
+##### chmod file does not have metadata (default)
 
 Chmod will only have one effect, if you remove all the write attributes of a file then the 'read only' attribute on the Windows file will be set, since this is the same behaviour as CIFS (Common Internet File System) which is the SMB (Server Message Block) client in Linux.
 
-##### The file has metadata
+##### chmod file has metadata
```
Comment (craigloewen-msft) [inline WSL/file-permissions.md:60]: Same comment as above
```suggestion
##### The file has metadata

```

File: WSL/index.md [hunk 9]
```diff
@@ -1,37 +1,39 @@
 ---
 title: An overview on the the Windows Subsystem for Linux
-description: Learn about the Windows Subsystem for Linux, the different versions and ways you can use them.
+description: Explore the Windows Subsystem for Linux documentation.
```
Comment (craigloewen-msft) [inline WSL/index.md:3]: Was this file moved out of the TOC? I can't seem to access it anymore.

File: WSL/toc.yml [hunk 10]
```diff
@@ -1,43 +1,37 @@
+- name: WSL Documentation
+  href: index.md
 - name: Overview
-  href: overview.md
```
Comment (craigloewen-msft) [inline WSL/toc.yml:2]: We removed overview.md . Not sure if that's on purpose or if we do want it anywhere.

File: WSL/wsl-config.md [hunk 11]
```diff
@@ -2,28 +2,24 @@
...
 
-There are many ways to run Linux with the Windows Subsystem for Linux.
+There are several ways to run a Linux distribution with WSL once it's [installed](install-win10.md).
 
-1. `[distro]`, for example `ubuntu`
-1. `wsl.exe` or `bash.exe`
-1. `wsl [command]` or `bash -c [command]`
+1. Open your Linux distribution by visiting the Windows Start menu and typing the name of your installed distributions. For example: "Ubuntu".
+2. From Windows Command Prompt or PowerShell, enter the name of your installed distribution. For example: `ubuntu`
+3. From Windows Command Prompt or PowerShell, to open your default Linux distribution inside your current command line, enter: `wsl.exe` or `bash.exe`
```
Comment (craigloewen-msft) [inline WSL/wsl-config.md:17]: We're trying to move users to using `wsl.exe`, `bash.exe` is old and doesn't have as many features.

```suggestion
3. From Windows Command Prompt or PowerShell, to open your default Linux distribution inside your current command line, enter: `wsl.exe`.
```

File: WSL/wsl-config.md [hunk 12]
```diff
@@ -2,28 +2,24 @@
...
+4. From Windows Command Prompt or PowerShell, to open your default Linux distribution inside your current command line, enter:`wsl [command]` or `bash -c [command]`
```
Comment (craigloewen-msft) [inline WSL/wsl-config.md:18]:
```suggestion
4. From Windows Command Prompt or PowerShell, to open your default Linux distribution inside your current command line, enter:`wsl [command]`.
```

**Comment by craigloewen-msft** [comment]
Only blocking changes are:
* Are we sure we want to remove overview.md? (Fine either way just wanted to call it out)
* There's a table in about.md that isn't filled out that needs to be addressed before we can merge.

File: WSL/file-permissions.md (hunk 8 above)
Comment (mattwojo) [reply WSL/file-permissions.md:60]: We can't reuse the same header over and over like this

File: WSL/index.md (hunk 9 above)
Comment (mattwojo) [reply WSL/index.md:3]: Which file?

File: WSL/toc.yml (hunk 10 above)
Comment (mattwojo) [reply WSL/toc.yml:2]: On purpose

**Pull request merged**